yellowbrick/features/pca.py

```diff
@@ -108,63 +128,87 @@ def __init__(
         self,
         ax=None,
         features=None,
+        classes=None,
         scale=True,
-        proj_dim=2,
+        projection=2,
         proj_features=False,
-        color=None,
-        colormap=palettes.DEFAULT_SEQUENCE,
+        colors=None,
+        colormap=None,
         alpha=0.75,
         random_state=None,
-        colorbar=False,
+        colorbar=True,
         heatmap=False,
         **kwargs
     ):
-        super(PCADecomposition, self).__init__(ax=ax, features=features, **kwargs)
-
-        if proj_dim not in (2, 3):
-            raise YellowbrickValueError("proj_dim object is not 2 or 3.")
+        super(PCADecomposition, self).__init__(
+            ax=ax,
+            features=features,
+            classes=classes,
+            colors=colors,
+            colormap=colormap,
+            projection=projection,
+            alpha=alpha,
+            colorbar=colorbar,
+            **kwargs
+        )
 
         # Data Parameters
         self.scale = scale
-        self.proj_dim = proj_dim
         self.proj_features = proj_features
 
         # Create the PCA transformer
         self.pca_transformer = Pipeline(
             [
                 ("scale", StandardScaler(with_std=self.scale)),
-                ("pca", PCA(self.proj_dim, random_state=random_state)),
+                ("pca", PCA(self.projection, random_state=random_state)),
             ]
         )
         self.alpha = alpha
 
         # Visual Parameters
-        self.color = color
-        self.colormap = colormap
-        self.colorbar = colorbar
         self.heatmap = heatmap
 
-        self.uax, self.lax = None, None
+        self._uax, self._lax = None, None
 
-        if self.proj_dim == 3 and (self.heatmap or self.colorbar):
+        # No heatmap can be drawn with 3d plots as they do not have permit axes
+        # division.
+        if self.projection == 3 and self.heatmap:
             raise YellowbrickValueError(
                 "heatmap and colorbar are not compatible with 3d projections"
             )
 
-        if self.heatmap or self.colorbar:
-            self._layout()
+    @property
+    def uax(self):
```

Comment:
I can't see this in the `projection.py` file but could you confirm we also have `cax` property?

Reply:
Also, why `uax`? Underneath axes? Why `lax`?

Reply:
Yes, we do have a property for cax.

Reply:
`uax` refers to upper axes and `lax` to lower axes. These names were suggested by Rebecca in the PR [here](https://github.com/DistrictDataLabs/yellowbrick/pull/884#issuecomment-502471237).

Reply:
Awesome, thanks! (Might be good to add a comment about that in the next PR)